examples/dpli_wpli_pli.py

```diff
@@ -0,0 +1,381 @@
+'''
+=============================
+Comparing PLI, wPLI, and dPLI
+=============================
+
+This example demonstrates the different connectivity information captured by
+the phase lag index (PLI) :footcite:`StamEtAl2007`, weighted phase lag index
+(wPLI) :footcite:`VinckEtAl2011`, and directed phase lag index (dPLI)
+:footcite:`StamEtAl2012` on simulated data.
+'''
+
+# Authors: Kenji Marshall <kenji.marshall99@gmail.com>
+#          Charlotte Maschke <charlotte.maschke@mail.mcgill.ca>
+#          Stefanie Blain-Moraes <stefanie.blain-moraes@mcgill.ca>
+#
+# License: BSD (3-clause)
+
+
+import mne
+import numpy as np
+import matplotlib.pyplot as plt
+
+from mne_connectivity import spectral_connectivity_epochs
+from mne.datasets import sample
+
+###############################################################################
+# Background
+# ----------
+#
+# The formulae for PLI, wPLI, and dPLI are given below. In these equations,
+# :math:`X_{ij}` is the cross-spectral density (CSD) between two signals
+# :math:`i, j`. Importantly, the imaginary component of the CSD is maximal
+# when the signals have a phase difference given by :math:`k\pi+\frac{\pi}{2}`,
+# and is :math:`0` when the phase difference is given by :math:`k\pi` (where
+# :math:`k \in \mathbb{Z}`). This property provides protection against
+# recognizing volume conduction effects as connectivity, and is the backbone
+# for these methods :footcite:`VinckEtAl2011`.
+#
+# :math:`PLI = |E[sgn(\mathcal{I}(X_{ij}))]|` :footcite:`StamEtAl2007`
+#
+# :math:`wPLI = \frac{|E[\mathcal{I}(X_{ij})]|}{E[|\mathcal{I}(X_{ij})|]}`
+# :footcite:`VinckEtAl2011`
+#
+# :math:`dPLI = E[\mathcal{H}(\mathcal{I}(X_{ij}))]` :footcite:`StamEtAl2012`
```

Comment:
Perhaps it would be beneficial for the reader to define what $\mathcal{I}$ and $\mathcal{H}$ is and $sgn$.